modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/TableCell.tsx

```diff
@@ -157,14 +157,25 @@ function TableCell({
 		if (!loading && view.contentRendererModuleURL && !dataRenderer) {
 			setLoading(true);
 
-			getDataRendererByURL(view.contentRendererModuleURL)
+			getDataRendererByURL(
+				view.contentRendererModuleURL,
+				view.contentRendererClientExtension
+					? 'clientExtension'
+					: 'internal'
+			)
 				.then((dataRenderer) => {
 					setDataRenderer(() => dataRenderer);
 
 					setLoading(false);
 				})
-				.catch(() => {
-					setDataRenderer(() => null);
+				.catch((error) => {
+					// eslint-disable-next-line no-console
+					console.log(
+						`Unable to load FDS cell renderer at ${view.contentRendererModuleURL}:`,
+						error
+					);
```

Comment:
We can use `console.error` here. With it, there is no need for linter exception.

Reply:
True. I thought that the stack trace of the original error could be lost but it turns out `console.error` handles it correctly (I've checked it) so I'll change it :+1:

Reply:
😱 you've found out how to get around our eslint rules...

Reply:
113 usages of `console.error(` in DXP :sweat_smile: